starboard/shared/uwp/media_is_video_supported.cc

```diff
@@ -144,7 +144,7 @@ class XboxVideoCapabilities {
                                             60);
         hw_decoder_capabilities_.AddHdrRule(kSbMediaVideoCodecAv1, 3840, 2160,
                                             30);
-        hw_decoder_capabilities_.AddHdrRule(kSbMediaVideoCodecAv1, 1920, 1080,
+        hw_decoder_capabilities_.AddHdrRule(kSbMediaVideoCodecAv1, 3840, 2160,
```

Comment:
There were just tests resolutions. All tests have shown about zero drops in this mode. Despite of this we have to return resolution to 2K because of overheat danger in 4K mode. Will be fixed in next commit.

Reply:
We mentioned overheating problem here:
https://docs.google.com/spreadsheets/d/13BZFYe27ldewqI12PYcKn6_8Lkm5m5mWS0n33WWysq8/edit#gid=585588393
in the table Maximum supported resolution.